### DISCUSSION TITLE
False Data FW2.14

### DISCUSSION THREAD

**Discussion by ogglobi** [mampfes/ha_freeair_connect#11] [Q&A]
Hi there.
Sadly I have a problem with one of my FreeAir100 devices that i updated to 2.14.
All the data im getting is pretty much false ... can you maybe look into this ?
I tried removing the device multiple times without success - always the same false data despite showing correct at freeair-connect.de

**Comment by mampfes** [comment]
Hi, I don't own such a device, therefore I can't debug this at the moment. What you could do (if you have some programming knowledge) is to log the response of the HTTP request and post it here.

**Comment by mampfes** [comment]
Continued in #13
